ui/src/app/pages/admin/admin-user-profile.tsx

```diff
@@ -192,13 +205,15 @@ export const AdminUserProfile = (spinnerProps: WithSpinnerOverlayProps) => {
     }
   }
 
-  const changed = (field): boolean => !!getUpdatedProfileValue(oldProfile, updatedProfile, field);
+  const changed = (attributePath): boolean => !!getUpdatedProfileValue(oldProfile, updatedProfile, attributePath);
```

Comment:
why attributepath and not field?

Reply:
not sure if path is the right word here, but it now represents an array of fields (I'll add the type!) because it can now be used to compare deeper within the object, like on line 213: 
`changed(['verifiedInstitutionalAffiliation', 'institutionShortName'])`

Reply:
All the changes from L183 to L216 are part of a bug fix which is not strictly related to the styling change in this PR.  It's in a separate commit so I can easily split it out into a different PR if you prefer

Reply:
I did this -> #6116